client/blocks/product-selector/index.jsx

```diff
@@ -81,6 +81,17 @@ export class ProductSelector extends Component {
 		} );
 	}
 
+	getDescriptionByProduct( product ) {
+		const { description, optionDescriptions } = product;
+		const purchase = this.getPurchaseByProduct( product );
+
+		if ( ! purchase || ! optionDescriptions ) {
+			return description;
+		}
+
+		return optionDescriptions[ purchase.productSlug ];
```

Comment:
In theory, we could have forgotten to define an option description for the current purchase product slug, so let's make sure that we return the default description in that case too. Something like:

```
if ( ! optionDescriptions[ purchase.productSlug ] ) {
	return description;
}
```

Reply:
I'm going to add this check to the conditional statement above.